mjpc/tasks/humanoid/stand/task.cc

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 DeepMind Technologies Limited
```

Comment:
No changes here. This is copy-pasted from the old `mjpc/tasks/humanoid/walk/task.cc`. I'll open a separate PR for these unless you prefer keeping all these in the same file.